shlr/squashfs/src/Makefile

```diff
@@ -97,11 +97,11 @@ INCLUDEDIR = -I.
 INSTALL_DIR = /usr/local/bin
 CFLAGS+=-I/usr/local/include -g
 
-MKSQUASHFS_OBJS = mksquashfs.o read_fs.o sort.o swap.o pseudo.o compressor.o \
+MKSQUASHFS_OBJS = mksquashfs.o read_fs.o sort.o pseudo.o compressor.o \
 	action.o
 
 UNSQUASHFS_OBJS = unsquashfs.o unsquash-1.o unsquash-2.o unsquash-3.o \
```

Comment:
this swap.o refers to its own swap.c implementation (shlr/squashfs/src/swap.c) so i would maintain swap.o here